node_package/tests/bootstrap.config.test.js

```
@@ -106,18 +106,23 @@ test('createConfig uses default config if user config is not provided', assert =
 });
 
 test('createConfig uses user config as expected', assert => {
+  const testConfigsDir = path.resolve(__dirname, './test_configs');
   const expectedResult = {
     appStyles: undefined,
-    bootstrapCustomizations:
-      '/home/ubuntu/workspace/node_package/tests' +
-      'test_configs/path/to/bootstrap/customizations.scss',
+    bootstrapCustomizations: path.resolve(
+      testConfigsDir,
+      './path/to/bootstrap/customizations.scss'
```

ESLint

[error] Insert `,`

(prettier/prettier)